.github/workflows/docker.yml

```diff
@@ -0,0 +1,22 @@
+name: Docker Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  docker-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          set -x -e
+          echo "Docker release ${{ github.event.release.tag_name }}:"
+          release=${{ github.event.release.tag_name }}
+          version=${release:1}
+          export DOCKER_LOGIN=${{ secrets.DOCKERHUB_USERNAME }}
+          export DOCKER_PASSWORD=${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
These should be done as environment variables

```yaml
jobs:
  docker-release:
    env:
      DOCKER_LOGIN: ${{ secrets.DOCKERHUB_USERNAME }}
...
```

Reply:
@SuperQ  Done.